src/registrar/tests/test_forms.py

```diff
@@ -74,6 +78,113 @@ def test_requested_domain_ending_dotcom_invalid(self):
             ["Enter the .gov domain you want without any periods."],
         )
 
+    def test_requested_domain_errors_consistent(self):
```

Comment:
These can be separated out of their for loop into separate test cases (rather than subTest) but I really don't think it makes sense to do here, given the repeatability of each test and the fact that we could expand our errors in the future. In my opinion this an appropriate use of subTest.